src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -398,7 +398,8 @@ private boolean moveRubricCol(int qnNumber, int colNumber, boolean isMoveLeft) {
         WebElement moveColButton = browser.driver.findElement(By.id(elemId));
 
         if (moveColButton.getAttribute("disabled") == null) {
-            moveColButton.click();
+            waitForElementToBeClickable(moveColButton);
```

Comment:
I think it's only required if the buttons dynamically change state and might be expected to not be clickable at that point in the test. Since the non-disabled buttons are always clickable, I feel it can be removed.

Defensive coding is more about using assertions and checking for null, and is more relevant in production code. For UI tests, we shouldn't introduce too many waits. It's good to quickly fail the test if the button is somehow not clickable at this point.

Also, according to Selenium code comments for `ExpectedConditions.elementToBeClickable`, an element is clickable if it is both visible and enabled. The code for determining visibility and enablement calls `isDisplayed()` and `isEnabled()` on the element. Taking cue from their code, we can replace the condition `moveColButton.getAttribute("disabled") == null` with `moveColButton.isEnabled()` for more readable code.